slider/position.py

```diff
@@ -16,3 +16,21 @@ class Position(namedtuple('Position', 'x y')):
     The visible region of the osu! standard playfield is [0, 512] by [0, 384].
     Positions may fall outside of this range for slider curve control points.
     """
+
+class Point(namedtuple('Point', 'x y offset')):
```

Comment:
That sounds like a good idea except in certain cases. For example, the control points (which are ironically usually called points not positions) really don't have a time and are currently Positions. Making them Points might be less clear to people and have them misunderstand thinking that the control points should have a time.